charts/backstage/values.yaml

```diff
@@ -308,3 +308,18 @@ serviceAccount:
 
   # -- Auto-mount the service account token in the pod
   automountServiceAccountToken: true
+
+# ServiceMonitor configuration
+serviceMonitor:
```

Comment:
perhaps we could wrap all of these properties under `metrics.serviceMonitor`.
I've seen it as a common pattern in many charts

Reply:
I agree, that seems to be a non-written-yet-established pattern across other charts. :slightly_smiling_face:

Reply:
Nice, I'll adopt that! Do you have any example charts/prior art that use this pattern?